docs/CHANGELOG.asciidoc

```diff
@@ -38,6 +38,32 @@ Review important information about the {kib} 8.4.x releases.
 
 Review the following information about the {kib} 8.4.0 release.
 
+[float]
+[[known-issue-8.4.0]]
+=== Known issues
+
+If you have alerting rules with configured snooze notifications, do not upgrade {kib} to 8.4.0.
```

Comment:
```suggestion
If you have alerting rules configured to snooze notifications, do not upgrade {kib} to 8.4.0.
```

Reply:
Can you check what I changed it to and let me know if you like it?

Reply:
Yes, that sounds good!